TestsCommon/GoKnownIssues.cs

```diff
@@ -0,0 +1,250 @@
+                { "create-accessreviewscheduledefinition-allusers-m365-aadrole-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "create-accessreviewhistorydefinition-from--go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "delete-activitybasedtimeoutpolicy-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-activitybasedtimeoutpolicy-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "create-activitybasedtimeoutpolicy-from-activitybasedtimeoutpolicies-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "update-activitybasedtimeoutpolicy-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "update-adminconsentrequestpolicy-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "delete-scopedrolemember-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "delete-administrativeunit-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-scopedrolemember-1-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-administrativeunit-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-scopedrolemember-2-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "post-administrativeunits-members-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "create-scopedrolemembership-from-administrativeunit-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "update-administrativeunit-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "delete-agreement-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-agreement-files-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "update-agreement-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-alert-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "update-alert-1-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "update-alert-2-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "appconsentrequest-filterbycurrentuser-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-appconsentrequest-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "application-addkey-1-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "application-addkey-2-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "application-addpassword-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "delete-application-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "application-delta-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-application-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-extensionproperties-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "application-get-owners-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-web-count-go-V1-compiles", NeedsAnalysisKnownIssue},
+                { "get-web-count-go-V1-compiles", NeedsAnalysisKnownIssue},
```

Comment:
duplicate entries here fails `dotnet test` call.
```suggestion
                { "get-web-count-go-V1-compiles", NeedsAnalysisKnownIssue},
```

Reply:
committing this to unblock the test discovery.